sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java

```diff
@@ -530,6 +530,12 @@ private CaseDataDto findCase(String uuid) {
 
 	protected CaseDataDto saveCase(CaseDataDto cazeDto) {
 
+		if (cazeDto.getReInfection() == YesNoUnknown.NO || cazeDto.getReInfection() == YesNoUnknown.UNKNOWN) {
```

Comment:
Can be checked directly if cazeDto.getReInfection() != YesNoUnknown.Yes instead of the above 2 conditions